impl/httpCppRest/NHttpClientCppRest.cpp

```diff
@@ -92,6 +92,17 @@ void NHttpClientCppRest::setBaseUri(const std::string& uri)
     _client = std::make_unique<http_client>(http_client(FROM_STD_STR(uri)));
 }
 
+
+
+void NHttpClientCppRest::setTimeout(int seconds)
```

Comment:
1. seconds might be too coarse
2. use explicit std::chrono:: type for duration should be more ergonomic